db/migrate/20190627083110_change_token_value_type.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeTokenValueType < ActiveRecord::Migration[5.2]
+  def up
+    unless column_exists? :tokens, :value, :text
```

Comment:
just so we don't run this again on new installs, only on upgrades that had already executed the previous migration

Reply:
Ok fine, I'd probably drop a comment as well why we have this (MySQL failed migrations).